src/OSPSuite.SimModelSolver_CVODES/src/SimModelSolver_CVODES.cpp

```diff
@@ -303,6 +302,10 @@ int SimModelSolver_CVODES::PerformSolverStep(double tout, double* y, double** yS
       // perform next solver step
       iResultflag = CVode(_cvodeMem, tout, _solution, &tret, CV_ONE_STEP);
 
+   	_step++;
```

Comment:
I don't see where `_step `is **initially** set to 0

Reply:
It's in FillSolverOptions

Reply:
ah ok, see now